osf/migrations/0146_pagecounter_id.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-11-10 23:02
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import osf.models.base
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0145_pagecounter_data'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='pagecounter',
+            name='_id',
+            field=models.CharField(db_index=True, default=osf.models.base.generate_object_id, max_length=50, unique=True),
```

Comment:
Migration to use generate_object_id to make the _id field instead of building it like: 'action:node_id:file_id:version_idx'. I left existing _ids as is, but don't know if we should migrate them as well (are we currently exposing this anywhere?) Also if we wouldn't ever expose this, perhaps the _id field could be removed entirely at this point.

Reply:
My gut says if we're making new `PageCounter`s with new `_id`, we should migrate all of em for consistency and to help future-selves from being confused. But you bring up a good point - I don't see a need really for an object_id as they're never being used like in an API detail endpoint or something like that - as long as we don't query on them anymore I suppose!

Reply:
Yeah, I'm not sure which way to go, I'll have to get Steve to weigh in.

Reply:
Discussed w/ Steve in-person, I'm going to revert this and keep constructing long id as we were previously (a mash-up of the four required fields), even though we don't need to keep querying this way. This will keep things consistent for a bit, while we make sure we don't need to revert back to querying on _id for some reason.  After some time has elapsed and we're sure this is stable, then we can make a migration to turn everything into long ids.  Thanks for looking out @erinspace!